appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: '{build}'
+pull_requests:
+  do_not_increment_build_number: true
+os: Visual Studio 2017
+shallow_clone: false
+
+environment:
+  MSYS2_PATH_TYPE: inherit
+  CHERE_INVOKING: enabled_from_arguments
+
+install:
+- c:\msys64\usr\bin\bash -lc "pacman --noconfirm -S curl unzip"
```

Comment:
You can skip that part if preinstall called in script (line 32).

Reply:
Cleaned it up.